model/src/pyrenew/model/rtinfectionsrenewal.py

```diff
@@ -11,7 +12,22 @@
     ["Rt", "latent", "observed"],
     defaults=[None, None, None],
 )
-"""Output from RtInfectionsRenewalModel.sample()"""
+RtInfectionsRenewalSample.__doc__ = """
+A container for holding the output from RtInfectionsRenewalModel.sample().
+
+Attributes
+----------
+Rt : float or None
+    The reproduction number over time. Defaults to None.
+latent : ArrayLike or None
+    The estimated latent infections. Defaults to None.
+observed : ArrayLike or None
+    The observed infections. Defaults to None.
+
+Notes
+-----
+Defaulting to None often indicates use for simulation, rather than for fitting.
```

Comment:
As above, this is not really true in the context of these return structs, so I would suggest removing it. The point is more relevant to setting `obs=None` (or the equivalent) in a `sample()` call.

```suggestion